website/docs/command-reference/registry.md

```diff
@@ -0,0 +1,88 @@
+---
+title: odo registry
+sidebar_position: 4
+---
+# odo registry
+
+odo uses the portable *devfile* format to describe the components. odo can connect to various devfile registries to download devfiles for different languages and frameworks.
+
+You can connect to available public devfile registries, or you can install your own [Secure Registry](/docs/architecture/secure-registry).
+
+You can use the `odo registry` command to manage the registries used by odo to retrieve devfile information.
+
+## Listing the registries
+
+You can use the following command to list the registries currently contacted by odo:
+
+```
+odo registry list
+```
+
+For example:
+
+```
+$ odo registry list
+NAME                       URL                             SECURE
+DefaultDevfileRegistry     https://registry.devfile.io     No
+```
+
+After installation, the odo registry will contain the default devfile registry, provided by the `devfile.io` project. 
```

Comment:
`devfile.io` This does not appear as a link(URL) on the site, it appears as an inline code.